README.md

```diff
@@ -100,51 +79,102 @@ The message adapter library contains a logging class `AdapterLogger` that standa
+
+    ```xml
+    <dependency>
+      <groupId>gov.nasa.earthdata</groupId>
+      <artifactId>cumulus-message-adapter</artifactId>
+      <version>1.2.12</version>
+    </dependency>
+    ```
+
+3. From the task project root, install the locally built package as a dependency and copy it into place (making sure that the version number is correct in `-Dfile` and `-Dversion`):
+
+    ```shell
+    mvn install:install-file \
+      -Dfile=/path/to/cumulus-message-adapter-java/message_parser/target/cumulus-message-adapter-1.2.12.jar \
+      -DgroupId=gov.nasa.earthdata \
+      -DartifactId=cumulus-message-adapter \
+      -Dversion=1.2.12 \
+      -Dpackaging=jar \
+      -DgeneratePom=true
+    mvn clean dependency:copy-dependencies
+    ```
+
+Running your task code (locally or when packaged and deployed) should now use the locally built CMA Java client package.
 
 ### Integration Tests
 
-Integration tests are located in the test folder in ```MessageParserTest.java```. By default, the latest [Cumulus Message Adapter](https://github.com/cumulus-nasa/cumulus-message-adapter) will be downloaded and used for the tests. To build and run the tests using the latest version of `Cumulus Message Adapter', run
+Integration tests are located in the test folder in `MessageParserTest.java`. By default, the latest [Cumulus Message Adapter](https://github.com/cumulus-nasa/cumulus-message-adapter) will be downloaded and used for the tests.
+
+To build and run the tests using the latest version of Cumulus Message Adapter, run:
 
-```mvn -B test```
+```shell
+cd message_parser
+mvn -B test
+```
 
-To build and run the tests using a different version of `Cumulus Message Adapter`, run
+To build and run the tests using a different version of the Cumulus Message Adapter, run
 
-```MESSAGE_ADAPTER_VERSION=vx.x.xx mvn -B test```
+```shell
+MESSAGE_ADAPTER_VERSION=vx.x.xx mvn -B test
+```
 
 ### Running the example task
 
 Follow the installation instructions above for the example task.
 
-If updating the version of the message parser, make sure to update the pom.xml in the task code. To build the task with this dependency, run:
+If updating the version of the message parser, make sure to update the `pom.xml` in the task code. To build the task with this dependency, run:
 
-```mvn clean install -U```
-
-then ```mvn -B package```
+```shell
+cd task/
+mvn clean install -U
+mvn -B package
+```
 
-## Why?
+## Benefits of the Cumulus Message Adapter
 
 This approach has a few major advantages:
 
 1. It explicitly prevents tasks from making assumptions about data structures
    like `meta` and `cumulus_meta` that are owned internally and may therefore
    be broken in future updates. To gain access to fields in these structures,
    tasks must be passed the data explicitly in the workflow configuration.
-1. It provides clearer ownership of the various data structures. Operators own
+2. It provides clearer ownership of the various data structures. Operators own
```

Comment:
The use of `1.` on _every_ numbered item is to allow auto-numbering, specifically so you don't have to supply the correct numbers, which is helpful when adding, removing, or reordering items. The generated HTML will include `<ol>` and `<li>` tags to automatically number items.

Reply:
Yeah, it may be a minor concern, but I feel like using `1` for every item isn't very clear if you're just reading the source code directly.

Reply:
That's cool. Just wanted to make sure you knew that using all `1.`s is valid. In fact, random numbering is valid too, as long as the first item is numbered `1.` I'm fine with your change here.